CONTRIBUTING.md

```diff
@@ -238,6 +238,19 @@ need to "save" the changes.
    - This will modify the appropriate patches based on your commits.
 1. PR your modified patch file(s) back to this repository.
 
+## Rebasing PRs
+Steps to rebase a pr to include the latest changes from `master`.
+
+1. Pull latest changes from upstream's master; `git checkout master && git pull`.
```

Comment:
This assumes that upstream is `origin`, which is not always the case (rarely for me, in fact). Make it clear.

Reply:
Does it? Upstream would always refer to the Paper repository for a fork, since that's literally the definition of upstream in this case. Origin would be the fork itself in this case.

Reply:
In all forks I do, `upstream` is the repository I forked, and `origin` is my own fork repository. This is not an uncommon setup, so what we assume here should either be explicit, or it should explicitly define what remote it fetches from (i.e. `git checkout origin/master`).

Reply:
I agree that we should make it clearer, however, if we do so, upstream should be PaperMC/Paper and origin Your/Paper. I think that's what the majority uses.